guides/source/active_record_migrations.md

```diff
@@ -917,35 +917,27 @@ Schema Dumping and You
 ### What are Schema Files for?
 
 Migrations, mighty as they may be, are not the authoritative source for your
-database schema. That role falls to either `db/schema.rb` or an SQL file which
-Active Record generates by examining the database. They are not designed to be
-edited, they just represent the current state of the database.
+database schema. Your database remains the authoritative source. By default,
+Rails generates `db/schema.rb` which attempts to capture the current state of
```

Comment:
It is missing the SQL file here.

Reply:
I thought it was sufficient to cover the default here and leave the specifics of the two formats to the section dedicated to them, but I can also re-add reference to `structure.sql` being a possibility here if needed.

Reply:
:+1: that make sense.